FWCore/Utilities/interface/IndexSet.h

```diff
@@ -0,0 +1,42 @@
+#ifndef FWCore_Utilities_IndexSet_h
+#define FWCore_Utilities_IndexSet_h
```

Comment:
I'm not sure if this is the best placement, but there are similar kind of utilities there already.